golem_messages/message/base.py

```diff
@@ -241,7 +241,8 @@ def __eq__(self, obj):
         return (

```

Comment:
Can we add comment/docstring here to explicitly state that we're ignoring `.encrypted` in comparison?

Reply:
@jiivan sure! :)